templates/components/products/product-view.html

```diff
@@ -150,24 +150,31 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
     </section>
 
     <section class="productView-images" data-image-gallery>
+        {{!--
+            Note that these image sizes are coupled to image sizes used in /assets/js/theme/common/product-details.js
+            for variant/rule image replacement
+        --}}
         <figure class="productView-image"
                 data-image-gallery-main
                 {{#if product.main_image}}
-                data-zoom-image="{{getImage product.main_image 'zoom_size' (cdn theme_settings.default_image_product)}}"
+                data-zoom-image="{{getImageSrcset product.main_image (cdn theme_settings.default_image_product) 1x="1280w" }}"
```

Comment:
Shouldn't we continue to use the `zoom_size` defined in the theme settings rather than hard code it here?

Reply:
Updated here and in product-details.js.